src/routes/controllers/project.controller.js

```diff
@@ -126,10 +133,40 @@ const deleteProject = async (req, res, next) => {
   }
 };
 
+const showServiceProject = async (req, res, next) => {
+  try {
+    const { projectId, secretKey: userSecretKey } = req.body;
+
+    if (!mongoose.isValidObjectId(projectId)) {
+      return next(createError(400, BAD_REQUEST));
+    }
+
+    const serviceProject = await Project
+      .findById(projectId)
+      .lean()
+      .populate()
+      .exec();
+
+    const validationResult = validateSecretKey(
+      userSecretKey,
+      serviceProject.secretKey,
+    );
+
+    if (validationResult) {
+      return res.json(serviceProject);
+    }
+
+    return res.json({ result: UNAUTHORIZED });
```

Comment:
```suggestion
    return res.status(401).json({ result: UNAUTHORIZED });
```
status 코드도 같이 보내는건 어떨까요?

Reply:
좋습니다! 반영할께요~